gtsfm/common/gtsfm_data.py

```diff
@@ -163,6 +170,11 @@ def select_largest_connected_component(self) -> "GtsfmData":
             return GtsfmData(self._number_images)
 
         cameras_in_largest_cc = graph_utils.get_nodes_in_largest_connected_component(camera_edges)
+        logger.info(
+            "Largest connected component contains {} of {} cameras".format(
+                len(cameras_in_largest_cc), self._number_images
```

Comment:
the original data itself might not have some cameras. I think you might want to print `len(self.get_valid_camera_indices())` here.

Reply:
@ayushbaid I didn't quite catch this -- we form the new data with `cameras_in_largest_cc`, so that should be equivalent to `get_valid_camera_indices())` right?